sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/OfflineStorageTests.cs

```diff
@@ -231,6 +231,34 @@ public void TransmissionStateIsSetToOpenOnFailedRequest()
             transmitter.Dispose();
         }
 
+        [Fact]
+        public void TelemetryIsStoredOfflineWhenTransmissionStateIsSetToOpen()
+        {
+            using var activity = CreateActivity("TestActivity");
+            var telemetryItem = CreateTelemetryItem(activity);
+            List<TelemetryItem> telemetryItems = new List<TelemetryItem>
+            {
+                telemetryItem
+            };
+
+            var mockResponse = new MockResponse(200).SetContent("Ok");
```

Comment:
is 200 the correct status code here?

Reply:
Yes, the idea is that it should even attempt to transmit even if the expected response was 200